metacatalog/ext/export_standard/schemas/iso19115/iso19115-2.j2

```diff
@@ -0,0 +1,412 @@
+                    {# which EntryGroup if there are more than one in Entry.associatedGroups? -> all of them? #}
+                    
+                    {# TODO: do we need citedResponsibleParty here? -> 'Identification of the contact for the resource.' #}
+                    <gmd:citedResponsibleParty>
+                        <gmd:CI_ResponsibleParty>
+                            <gmd:organisationName>
+                                <gco:CharacterString>NOAA's Office of Response and Restoration (OR&amp;R)</gco:CharacterString>
+                            </gmd:organisationName>
+                            <gmd:role>
+                                <gmd:CI_RoleCode codeList="http://www.isotc211.org/2005/resources/Codelist/gmxCodelists.xml#CI_RoleCode" codeListValue="originator" codeSpace="006">originator</gmd:CI_RoleCode>
+                            </gmd:role>                        
+                        </gmd:CI_ResponsibleParty>
+                    </gmd:citedResponsibleParty>
+                </gmd:CI_Citation>
+            </gmd:citation>
+            <gmd:abstract>
+                <gco:CharacterString>{{abstract}}</gco:CharacterString>
+            </gmd:abstract>
+
+            {# first author #}
+            <gmd:pointOfContact>
+                <gmd:CI_ResponsibleParty>
+                    <gmd:individualName>
+                        {# format: last_name, first_name #}
+                        <gco:CharacterString>{{author.last_name}}, {{author.first_name}}</gco:CharacterString>
+                    </gmd:individualName>
+                    <gmd:organisationName>
+                        <gco:CharacterString>{{author.organisation_name}}</gco:CharacterString>
+                    </gmd:organisationName>
+                    <gmd:role>
+                        <gmd:CI_RoleCode codeList="https://data.noaa.gov/resources/iso19139/schema/resources/Codelist/gmxCodelists.xml#CI_RoleCode" codeListValue="author">author</gmd:CI_RoleCode>
+                    </gmd:role>
+                </gmd:CI_ResponsibleParty>
+            </gmd:pointOfContact>
+
+            {# coauthors#}
+            {% if authors %}
+            {% for coauthor in authors %}
+            <gmd:pointOfContact>
+                <gmd:CI_ResponsibleParty>
+                    <gmd:individualName>
+                        {# format: last_name, first_name #}
+                        <gco:CharacterString>{{coauthor.last_name}}, {{coauthor.first_name}}</gco:CharacterString>
+                    </gmd:individualName>
+                    <gmd:organisationName>
+                        <gco:CharacterString>{{coauthor.organisation_name}}</gco:CharacterString>
+                    </gmd:organisationName>
+                    <gmd:role>
+                        <gmd:CI_RoleCode codeList="https://data.noaa.gov/resources/iso19139/schema/resources/Codelist/gmxCodelists.xml#CI_RoleCode" codeListValue="coAuthor">coAuthor</gmd:CI_RoleCode>
```

Comment:
We actually encoded the full `CI_RoleCode` into metacatalog and can use this information here accordingly.
Also, there are roles, that can be hard-coded into the template (The vforwter team as the distributor, I think),